apps/notifications/src/panel/state/ui/reducer.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * External dependencies
+ */
 import { combineReducers } from 'redux';
 
 import {
```

Comment:
Seems like we need an `Internal dependencies` annotation here?

Reply:
Good catch!
A good example of why this rule is forcing us to add comments that get out of sync with the code :)